File: 1-easy/problems/3701-compute-alternating-sum/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def alternatingSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sum(nums[i] for i in range(0, len(nums), 2))-sum(nums[i] for i in range(1, len(nums), 2))

File: 1-easy/problems/3701-compute-alternating-sum/solution.ts
function alternatingSum(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return sum(nums[i] for i in Array.from({length: 0, (nums}, (_, i) => i).length, 2))-sum(nums[i] for i in Array.from({length: 1, (nums}, (_, i) => i).length, 2));
}

export { alternatingSum };
